org.adempiere.base/src/org/compiere/model/MAuthorizationAccount.java

```diff
@@ -100,24 +100,44 @@ public String findRefreshToken() {
 	 * Synchronize information on other accounts with same email in the same credential
 	 */
 	public void syncOthers() {
-		final String script = "UPDATE AD_AuthorizationAccount "
-				+ "SET AccessToken=?, AccessTokenTimestamp=?, ExpireInSeconds=? "
+		final String clientSql =
+				"SELECT DISTINCT AD_Client_ID "
+				+ "FROM AD_AuthorizationAccount "
 				+ "WHERE AD_AuthorizationAccount_ID!=? "
 				+ "AND EMail=? "
 				+ "AND AD_AuthorizationCredential_ID=? "
 				+ "AND IsAuthorized='Y' "
 				+ "AND RefreshToken IS NOT NULL";
-		String accessToken = getAccessToken();
-		accessToken = SecureEngine.encrypt(accessToken, getAD_Client_ID());

```

Comment:
remove , getAD_Client_ID()

Reply:
Sorry @hieplq , I don't understand this comment

Reply:
sorry, wrong place
i mean this line int[] clientIds = DB.getIDsEx(get_TrxName(), clientSql, get_KeyColumns(),
need to be remove get_KeyColumns()

Reply:
Thanks @hieplq - committed